src/err/mod.rs

```diff
@@ -283,7 +283,7 @@ impl PyErr {
     ///     assert!(err.traceback(py).is_none());
     /// });
     /// ```
-    pub fn traceback<'py>(&'py self, py: Python<'py>) -> Option<&'py PyTraceback> {
+    pub fn traceback<'py>(&self, py: Python<'py>) -> Option<Bound<'py, PyTraceback>> {
```

Comment:
I think rather than changing this API we should keep it unchanged and add a new `traceback_bound` for now. Gosh, I'm going to be pleased when we can clean up again in 0.22 / 0.23 ;)

Reply:
sure, i guess I'll add add a deprecation warning to thin one than